docs/blocks/applying-effects.md

```diff
@@ -24,91 +24,79 @@ This tutorial aims to show how to apply status effects to entities as long as th
 
 ## Setup
 
-We will need to add a couple things to our code, first let's start with property that will hold `true` and `false` values:
+We will need to add a couple things to our code, first let's start with a state that will hold `true` when stood on, and `false` otherwise:
 
-<CodeHeader>BP/blocks/wither_block.json</CodeHeader>
+<CodeHeader>minecraft:block > description</CodeHeader>
```

Comment:
Before we always did BP/blocks/wither_block.json#description. Probably your method is better.